extensions/cal-com-share-meeting-links/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Cal.com Share Meeting Links Changelog
 
+## [Update] - 2024-07-26
```

Comment:
```suggestion
## [Update] - {PR_MERGE_DATE}
```

Reply:
Thanks for the suggestions. My compiler wasn't liking the Preferences type reference earlier but it looks fine now.